src/sentry/deletions/defaults/organizationintegration.py

```diff
@@ -8,21 +8,9 @@ def should_proceed(self, instance):
         return instance.status in {ObjectStatus.DELETION_IN_PROGRESS, ObjectStatus.PENDING_DELETION}
 
     def get_child_relations(self, instance):
-        from sentry.models import (
-            ExternalIssue,
-            Identity,
-            IntegrationExternalProject,
-            PagerDutyService,
-        )
+        from sentry.models import Identity, IntegrationExternalProject, PagerDutyService
 
         relations = [
-            ModelRelation(
```

Comment:
This is intentional.  First off, in hybrid cloud world, this direct cascade is not possible due to `OrganizationIntegration` / `Integration` belonging to control, while `ExternalIssue` belonging to region.  

Using hybrid cloud cascading is, possible, but more complex -- the fk is not the identifier of the join table, but the constituent columns.  This would require either
1.  tombstones carrying additional payloads or
2.  having hybrid cloud deletion logic do a multi region span out to find the additional data to perform the cascade.

Neither of these is ideal and carries additional performance complexity.

Given that this is the  ONLY case we're up against where this presents a problem, I propose the alternative: we don't cascade this deletion directly.  Instead, ExternalIssues are deleted upon Organization deletion via a future `HybridCloudForeignKey`.  

Can defend this position in greater detail if needed.

Reply:
>  I propose the alternative: we don't cascade this deletion directly. Instead, ExternalIssues are deleted upon Organization deletion via a future HybridCloudForeignKey.

This should be ok. Do we know what will happen to existing issue links when an integration is removed? Historically, those external issue links would be removed, but now they will not. Ideally the links still 'work', or are elided from the group links response.

ExternalIssue records will also be pruned when groups are deleted.

Reply:
```
        (integration, org_integration) = integration_service.get_organization_context(
            organization_id=organization_id, integration_id=integration_id
        )
        if not integration or not org_integration:
            return Response(status=404)
```

So essentially, even if they do exist, once the integration is gone it 404s.  They are inaccessible.